Libraries/Opc.Ua.Configuration/ApplicationConfigurationBuilder.cs

```diff
@@ -829,6 +928,69 @@ public IApplicationConfigurationBuilderExtension AddExtension<T>(XmlQualifiedNam
         }
         #endregion
 
+        #region Public Static Methods
+
+        /// <summary>
+        /// Create ApplicationCertificates from a PKI root.
+        /// </summary>
+        /// <param name="storeType">The cert store type: ex: "Directory"</param>
+        /// <param name="storePath">The PKI root.</param>
+        /// <param name="subjectName">The subject name.</param>
+        /// <returns>The application certificates.</returns>
+
+        public static CertificateIdentifierCollection CreateDefaultApplicationCertificates(
+            string subjectName,
+            string storeType = null,
+            string storePath = null)
+        {
+            CertificateIdentifierCollection certificateIdentifiers = new CertificateIdentifierCollection{
+                new CertificateIdentifier {
+                    StoreType = storeType,
+                    StorePath = storePath,
+                    SubjectName = subjectName,
+                    CertificateType = ObjectTypeIds.RsaSha256ApplicationCertificateType
+                },
+                new CertificateIdentifier {
```

Comment:
we should check if we can default to a single Nist/brainpool cert, if the higher and lower profile is supported.